api/registrations/serializers.py

```diff
@@ -68,6 +68,7 @@ class RegistrationSerializer(NodeSerializer):
         'withdrawn',
     ]
 
+    ia_url = ser.URLField(read_only=True, source='IA_url')
```

Comment:
this should be `source='ia_url'`, right?

Reply:
in fact, it looks `source` needs to be removed. 

> AssertionError /{version})/registrations/{node_id}/
It is redundant to specify `source='ia_url'` on field 'URLField' in serializer 'RegistrationDetai...